webapp/api/dashboard.py

```diff
@@ -71,6 +71,17 @@
 ])
 
 
+DASHBOARD_API_V2 = os.getenv(
+    'SNAPCRAFT_IO_API_V2',
```

Comment:
'DASHBOARD_API_V2'

Reply:
Ah, good catch - I renamed the var in code but not in env.